libbeat/cmd/instance/beat.go

```diff
@@ -430,7 +433,7 @@ func (b *Beat) TestConfig(bt beat.Creator) error {
 }
 
 // Setup registers ES index template, kibana dashboards, ml jobs and pipelines.
-func (b *Beat) Setup(bt beat.Creator, template, setupDashboards, machineLearning, pipelines bool) error {
+func (b *Beat) Setup(bt beat.Creator, template, setupDashboards, machineLearning, pipelines, policy bool) error {
```

Comment:
Agree that we should generalise this but not as part of this PR as I want to keep the scope of it as small as possible. It will also be backported to 6.x

Issue for refactoring can be found here: https://github.com/elastic/beats/issues/9342

Reply:
OK, to not do a scope creep here, I would have still preferred to move to a struct because we actually break the developer contract in 6.x by adding a new parameter.

Reply:
I took another look, we don't use this method outside of libbeat. So I am OK to break it.